LICENSE

```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
Are you using GPL2 because of compatibility with other licenses? On the whole, we use MIT for reasons of potential commercialisation (part of Sikt's remit that we may disagree with, but nevertheless exists)

Reply:
@anetteOlli happened to discover that our codebase must be derived from this repository somehow: https://github.com/makbn/JThumbnail

Do Sikt have anyone to consult about this? Licenses are complex stuff. At least now that our source code is publicly available/open sourced.

Reply:
I can't offer legal advice, but I am (as you @anetteOlli — hence my question) also lead to believe that if the work is derived (i.e. forked) from a GPL licensed source, then the GPL must be applied to the derived source code. On the whole, we use libraries as compiled source, which I have again been lead to believe means we avoid the issue. However, we can contact the legal department and ask their advice as @ketilaa points out. A side note here is that I would have liked some trustworthy automated scanning for legal issues